rpc_jobs/hw_multi_node_aio.yml

```diff
@@ -139,21 +145,22 @@
                   "DEPLOY_MAAS=no"
                   ]
               ) // deploy_sh
-              node(){{
-                maas.prepare(instance_name: instance_name)
-                maas.deploy()
-                maas.verify()
-                tempest.tempest("deploy1")
-                kibana.kibana(env.KIBANA_SELENIUM_BRANCH, "deploy1")
-                holland.holland()
-              }}
-              ssh_slave.destroy()
-              maas.entity_cleanup(instance_name: instance_name)
+              maas.prepare(instance_name: instance_name)
+              maas.deploy()
+              maas.verify()
+              tempest.tempest()
+              kibana.kibana(env.KIBANA_SELENIUM_BRANCH)
+              holland.holland()
             }} // hw node
           }} catch (e){{
               print e
               throw e
           }} finally {{
+              ssh_slave.destroy(instance_name)
+              maas.entity_cleanup(instance_name: instance_name)
+              if(deploy_node != null){{
+                ssh_slave.destroy(deploy_node)
```

Comment:
If the mnaio setup fails, then the deploy node as a slave won't exist, which causes this destroy to fail. It won't change the outcome of the build since it's already failed, but I thought if people saw that failure at a glance, they would think that was the original failure reason.